zkstack_cli/crates/zkstack/src/commands/server/run.rs

```diff
@@ -20,16 +21,7 @@ pub(super) async fn run_server(
     shell: &Shell,
 ) -> anyhow::Result<()> {
     let args = args.fill_values_with_prompt().await;
-
-    let ecosystem_config = EcosystemConfig::from_file(shell)?;
-    let chain = ecosystem_config
-        .load_current_chain()
-        .expect(MSG_CHAIN_NOT_FOUND_ERR);
-    let general_config = chain.get_general_config()?;
-    let api_config = general_config.api_config.context("Missing API config")?;
-
-    let rpc_port = api_config.web3_json_rpc.http_port.to_string();
-    let healthcheck_port = api_config.healthcheck.port.to_string();
+    let ports = EcosystemPortsScanner::scan(shell)?;
```

Comment:
this is relevant only for docker run, should we get it  so early?

Reply:
The problem is that we cannot get it within the `common` crate.